server/prisma/schema.prisma

```diff
@@ -10,66 +7,58 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-// TODO: ADD type attributes: https://www.prisma.io/docs/concepts/components/prisma-schema/data-model#defining-attributes
-// and define optional fields
-
-model User {
-  id              Int               @id @default(autoincrement())
-  email           String            @unique
-  email_verified  Boolean
-  username        String
-  picture_url     String
-  sub             String
-  created_at      DateTime          @default(now())
-  Comment         Comment[]
-  Post            Post[]
-  User_Space_Role User_Space_Role[]
-}
-
-model Post {
-  id         Int       @id @default(autoincrement())
-  title      String
-  tags       String
-  created_at DateTime  @default(now())
+model comment {
+  comment_id Int      @id @default(autoincrement())
   content    String
-  user       User      @relation(fields: [user_id], references: [id], onDelete: Cascade, onUpdate: Cascade)
-  user_id    Int
-  space      Space     @relation(fields: [space_id], references: [id], onDelete: Cascade, onUpdate: Cascade)
-  space_id   Int
-  Comment    Comment[]
+  created_at DateTime @db.Timestamp(6)
+  user_id    String   @db.VarChar(255)
+  post_id    Int
+  post       post     @relation(fields: [post_id], references: [post_id], onDelete: NoAction, onUpdate: NoAction)
+  user       user     @relation(fields: [user_id], references: [sub], onDelete: NoAction, onUpdate: NoAction)
 }
 
-model Space {
-  id              Int               @id @default(autoincrement())
-  name            String            @unique
-  description     String
-  created_at      DateTime          @default(now())
-  Post            Post[]
-  User_Space_Role User_Space_Role[]
+model post {
+  post_id     Int           @id @default(autoincrement())
+  space_id    Int
+  title       String
+  tags        String
+  content     String?
+  created_at  DateTime      @db.Timestamp(6)
+  space       space         @relation(fields: [space_id], references: [space_id], onDelete: NoAction, onUpdate: NoAction)
+  comment     comment[]
+  space_colab space_colab[]
```

Comment:
@thaistcosta do we not need a user_id here as well? Or we assuming that for now only the owner can post in its own space? sorry i forgot